CONTRIBUTING.md

```diff
@@ -81,6 +81,30 @@ cherry-picked into `book-v2` branch.
 
 Check the CI job after to do the Pull Request and then, click on in the `Details` of `netlify/kubebuilder/deploy-preview`
 
+## Understanding the versions
+
+|   Name	|   Example	|  Description |
+|---	|---	|---	|
+|  PROJECT version |  `V1`,`V2`,`V3` | After the Kubebuilder v3 introduces the plugins architecture to [Extensible CLI and Scaffolding Plugins](https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/extensible-cli-and-scaffolding-plugins-phase-1.md), the PROJECT version represents the version of the PROJECT file layout which are generated in the projects built with it.  |
+|  Releases version | `v2.2.0`, `v2.3.0`, `v2.3.1` | Represents the release tags of this project. See the [releases](https://github.com/kubernetes-sigs/kubebuilder/releases) page. |
+|  Plugin Versions | `go.kubebuilder.io/v2.0.0` | Represents the code source of the Plugin used to scaffold the projects. |
+
+Note that, we just need to bump a new PROJECT version if a breaking or significant change be introduce in the PROJECT file. Otherwise, it not requires be changed. And then, see that Releases and Plugin versions following the [semver](https://semver.org/) definition.  
+
+## Introducing changes in the scaffold files
+
+Changes in the scaffold files require a new Plugin version. Note that, if we delete and/or update a file or a content that is scaffolded by default, it will result in a breaking change and requires a `MAJOR` Plugin version. But, if we are adding a new content and/or file it probably will not be and breaking change. 
+
+For our sake let's try to simplify it and consider that any change that will break the expected behaviour of a project built with the previous `MINOR` Plugin versions will be a breaking change. 
```

Comment:
```suggestion
Changes in the scaffolded files require a new Plugin version. If we delete or update a file that is scaffolded by default, it's a breaking change and requires a `MAJOR` Plugin version.  If we add a new file, it may not be a breaking change.

More simply: any change that will break the expected behaviour of a project built with the previous `MINOR` Plugin versions is a breaking change to that plugin. 
```

Reply:
Done